.changeset/cuddly-hats-remember.md

```diff
@@ -0,0 +1,10 @@
+---
+"@pnpm/common-cli-options-help": minor
+"@pnpm/config": minor
+"@pnpm/filter-workspace-packages": minor
```

Comment:
This had major changes. Please create a separate changeset for the changes in `filter-workspace-packages` and describe the changes to its API